apps/native/app/src/index.tsx

```diff
@@ -61,11 +57,6 @@ async function startApp() {
     // Set the app root
     await Navigation.setRoot({ root })
 
-    if (isAndroid) {
-      // Handle initial notification on android
-      handleInitialNotificationAndroid()
```

Comment:
We handle the initial notification for both iOS and Android on the home screen instead